packages/frontend/src/components/Claim/ResultsAwaiting.tsx

```diff
@@ -1,9 +1,10 @@
-import { FormSectionWrapper } from 'src/components/Form/Form'
+import { FormHeading, FormSectionWrapper, FormText } from 'src/components/Form/Form'
 
 export const ResultsAwaiting = () => {
   return (
     <FormSectionWrapper>
-      <h2>Wait for results ⏳</h2>
+      <FormHeading>Wait for results ⏳</FormHeading>
+      <FormText>The bidding window has closed. Waiting for the organizers to settle the contest.</FormText>
```

Comment:
Similarly as in https://github.com/TrueFiEng/devcon-raffle/pull/141#issuecomment-1124841743, I'd increase the right padding a bit to make the text wrap earlier and look better.
![image](https://user-images.githubusercontent.com/3901896/168065082-7e381bd5-7f63-40fa-8cf9-c66f7e1a3610.png)